psutil/_psutil_posix.c

```diff
@@ -16,7 +16,7 @@
 #include <net/if.h>
 
 #ifdef PSUTIL_SUNOS10
-#include "arch/solaris10/ifaddrs.h"
+#include "arch/solaris/v10/ifaddrs.h"
```

Comment:
Can you test this on OpenSolaris x86? If not I will do but I'm not sure when (I have to reinstall virtualbox + work).